sources/Misc.cpp

```diff
@@ -125,6 +125,7 @@ uint64_t PLH::boundAllocLegacy(uint64_t start, uint64_t end, uint64_t size)
 
 		assert(mbi.RegionSize != 0);
 		if (mbi.State != MEM_FREE || mbi.RegionSize < size)

```

Comment:
Please add braces to this if expression. I believe just the addition would occur and the continue would always execute